crrExistingObjects.js

```diff
@@ -172,7 +176,7 @@ function _markObjectPending(bucket, key, versionId, storageClass,
                 destination,
                 storageClass,
                 role: Role,
-                storageType: '',
+                storageType: STORAGE_TYPE,
```

Comment:
I thought the issue was that the credentials used were not sufficient to access the admin routes. Also, can't storage type be populated by looking at the output from `getBucketReplication`?

Reply:
It appears that while the replication process will use the service credentials, the requests to the S3 and backbeat endpoints can be made with user-created storage account credentials. 

To get the storage type from information based on the replication configuration we’d need be able to query the location configurations of CloudServer to determine the mapping between location name and location type. I didn’t find a way to do this so figured this can be a simple solution, though it does add a bit more overhead for the user.

Reply:
After reviewing this again with @jonathan-gramain the changes applied are sufficient and working as expected.